jobs/search-term-data-validation/Dockerfile

```diff
@@ -2,9 +2,12 @@ FROM python:3
 MAINTAINER Chelsea Troy <ctroy@mozilla.com>
 
 WORKDIR /usr/app/src
```

Comment:
did you mean to replace this line?

Reply:
Tried to do this automated merge thing that did not work XD but currently taking a different approach anyway!